package.json

```diff
@@ -18,6 +18,7 @@
   "workspaces": [
     "./src/types",
     "./src/constants",
+    "./src/common-utils",
```

Comment:
Nit: I wish we could change the top level `src` to `modules` or `packages`... `src/<module>/src` is kind of confusing...